searchlib/src/vespa/searchlib/memoryindex/document_inverter_collection.h

```diff
@@ -0,0 +1,31 @@
+// Copyright Yahoo. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#pragma once
+
+#include <deque>
+#include <memory>
+#include <vector>
+
+namespace search::memoryindex {
+
+class DocumentInverter;
+class DocumentInverterContext;
+
+/*
+ * Class containing the document inverters used by a memory index.
+ */
+class DocumentInverterCollection {
```

Comment:
Consider adding a separate unit test for this class to test the management of free, inflight and active inverters.

Reply:
Unit test added.